File: Assignment 5/Activity 7.js
function main() {
    // This program converts Human years to Dog years.
    // Reference:
    // https://en.wikipedia.org/wiki/Aging_in_dogs
    var dogname;
    var humanyears;
    var dogyears;
    
    dogname = getdogname();
    dogyears = calculatedogyears();
    displayResult(dogname, dogyears);
}

function calculatedogyears() {
    var dogyears;
    var humanyears;
    
    window.alert("Enter dogs age is in human years");
    humanyears = window.prompt('Enter a value for humanyears');
    dogyears = humanyears * 7;
    
    return dogyears;
}

function displayResult(dogname, dogyears) {
    window.alert(dogname + " is " + dogyears + " Dog Years");
}

function getdogname() {
    var dogname;
    
    window.alert("Enter Dog's Name");
    dogname = window.prompt('Enter a value for dogname');
    
    return dogname;
}

File: Assignment 5/Activity 7.py
def calculatedogyears(humanyear):
    dogyears = humanyear * 7
    
    return dogyears

def displayResult(dogname, dogyears):
    print(dogname + " is " + str(dogyears) + " Dog Years")

def getdogname():
    print("Enter Dog's Name")
    dogname = input()
    
    return dogname

def gethumanyear():
    print("Enter dogs age is in human years")
    humanyear = float(input())
    
    return humanyear

# Main
# This program converts Human years to Dog years.
# Reference:
# https://en.wikipedia.org/wiki/Aging_in_dogs
dogname = getdogname()
humanyear = gethumanyear()
dogyears = calculatedogyears(humanyear)
displayResult(dogname, dogyears)
